js/area/signin/signin.js

```diff
@@ -1,6 +1,6 @@
 var $ = require('jquery'),
-	placeholder = require('../../../libs/jquery-placeholder/jquery.placeholder'),
-	vide = require('../../../libs/vide/dist/jquery.vide.min');
+	placeholder = require('../../../node_modules/jquery-placeholder/jquery.placeholder'),
```

Comment:
Now these are living in `node_modules` we shouldn't need to specify the full relative path – there's nothing wrong with doing so, but for convention's sake shall we... 
 ```placeholder = require(jquery-placeholder/jquery.placeholder')```
or maybe even...
 ```placeholder = require(jquery-placeholder')```
now we aren't using bower packages have the ability to specify an entry point in their `package.json`s, so we can afford to be less specific when resolving their files (not the case for all though).